assets/css/_route_properties_card.scss

```diff
@@ -0,0 +1,158 @@
+.m-route-properties-card {
+  width: 375px;
+}
+
+.m-route-properties-card .m-card__title {
+  display: flex;
+  flex-direction: row;
+  align-items: start;
+  padding: 0.5rem 0.5rem;
+  justify-content: space-between;
+  margin: 0;
+}
+
+.m-route-properties-card .route-title {
+  display: flex;
+  flex-direction: row;
+  gap: 0.5rem;
+  & h2 {
```

Comment:
**suggestion:** I don't think you need the `&` here or in the selector immediately below, since the descendant combinator is the default for nested SCSS selectors.

Reply:
Looks like this applies to a few other selectors throughout the file as well.

Reply:
Great, thank you! Also doing some reading up on BEM and will likely change some class names to follow the conventions more closely